model_compression_toolkit/exporter/model_wrapper/pytorch/builder/fully_quantized_model_builder.py

```diff
@@ -75,15 +76,22 @@ def get_activation_quantizer_holder(node: BaseNode, fw_impl) -> Callable:
         Returns:
             A PytorchActivationQuantizationHolder module for the node's activation quantization.
         """
-        _, activation_quantizers = fw_impl.get_inferable_quantizers(node)
         # Holder by definition uses a single quantizer for the activation quantization
         # thus we make sure this is the only possible case (unless it's a node we no activation
         # quantization, which in this case has an empty list).
-        if len(activation_quantizers) == 1:
-            return PytorchActivationQuantizationHolder(activation_quantizers[0])
-        Logger.critical(
-            f'PytorchActivationQuantizationHolder supports a single quantizer but {len(activation_quantizers)} quantizers '
-            f'were found for node {node}')
+
+        if node.is_quantization_preserving():
```

Comment:
This check is redundant, you already checked  `node.is_quantization_preserving()` before calling the function.
There is a little bit of confusion in the implementation of the different cases, I'll write a summarized comment separatly